### ISSUE TITLE
reduce output when entering multiple parent branch

### ISSUE THREAD

**Issue by charlierudolph** [Originate/git-town#609]
Current output
```
Feature branches can be branched directly off 
main or from other feature branches.
 
The former allows to develop and ship features completely independent of each other.
The latter allows to build on top of currently unshipped features.
 
  1: main
  2: feature-1
 
Please specify the parent branch of feature-2 by name or number (default: main): feature1
 
 
Feature branches can be branched directly off 
main or from other feature branches.
 
The former allows to develop and ship features completely independent of each other.
The latter allows to build on top of currently unshipped features.
 
  1: main
  2: feature-2
 
Please specify the parent branch of feature-1 by name or number (default: main): 
```

The explanation should probably only be given at most once per command.

```
Feature branches can be branched directly off 
main or from other feature branches.
 
The former allows to develop and ship features completely independent of each other.
The latter allows to build on top of currently unshipped features.
 
  1: main
  2: feature-1
 
Please specify the parent branch of feature-2 by name or number (default: main): feature1
 
  1: main
  2: feature-2
 
Please specify the parent branch of feature-1 by name or number (default: main): 
```
Or maybe even
```
Feature branches can be branched directly off 
main or from other feature branches.
 
The former allows to develop and ship features completely independent of each other.
The latter allows to build on top of currently unshipped features.
 
  1: main
  2: feature-1
  3: feature-2
 
Please specify the parent branch of feature-2 by name or number (default: main): feature1
Please specify the parent branch of feature-1 by name or number (default: main): 
```

**Comment by kevgo** [comment]
Good catch. I vote for version 2, because the branch list is different for each branch (I think). Should be an easy fix.

**Comment by charlierudolph** [comment]
I'm in favor of 3. The only difference in the branch list currently in not including the branch you are naming as the parent branch. And it doesn't look like we are guarding the case where you enter a branch as a parent of itself.

**Comment by kevgo** [comment]
I can get behind that. We should list all branches in the branch list then.

**Comment by charlierudolph** [comment]
Fixed in bd7b1f1b8231c3c92a10f80d11408639f45de8e3
